man/systemd-system.conf.xml

```diff
@@ -408,7 +408,24 @@
         <varname>LimitXXX=</varname> directives and they accept the same parameter syntax,
         see <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>
         for details. Note that these resource limits are only defaults
-        for units, they are not applied to the service manager process (i.e. PID 1) itself.</para></listitem>
+        for units, they are not applied to the service manager process (i.e. PID 1) itself.</para>
+
+        <para>Most of these settings are unset, which means the resource limits are inherited from the parent process.
+        However, the following have defaults:</para>
+        <itemizedlist>
+          <listitem><para><varname>DefaultLimitNOFILE=</varname> defaults to <literal>1024:524288</literal>.</para></listitem>
+
+          <listitem><para><varname>DefaultLimitMEMLOCK=</varname> defaults to the service manager's original
+          <varname>RLIMIT_MEMLOCK</varname>. Original refers to the initial <varname>RLIMIT_MEMLOCK</varname> applied
+          on the service manager, before systemd attempts to bump it higher.
+          </para></listitem>
```

Comment:
hmm, i'd drop this one. effectively this just means that systemd's payload process will get the exact same RLIMIT_NOFILE systemd itself got. THe fact that systemd bumps it up for itself is an implementation detail ultimately, since we try to make sure to never "leak" it to child processes.

it might be ok, to say it like that in an extra comment, just to make this easier to grok for people who  are confused by /proc/1/limits otherwise: "Note that the service manager internally increases RLIMIT_MEMLOCK for itself, however the limit is reverted to the original value for child processes forked off."